corefxlab.sln

```diff
@@ -131,7 +130,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Text.Json.Tests.Dyna
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Text.Primitives.Performance.Tests", "tests\System.Text.Primitives.Performance.Tests\System.Text.Primitives.Performance.Tests.csproj", "{04970622-6C7A-4FB4-814A-9BFE051A2E68}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.IO.Pipelines.Performance.Tests", "tests\System.IO.Pipelines.Performance.Tests\System.IO.Pipelines.Performance.Tests.csproj", "{E68CF803-CC9F-458A-B0C7-1C2E0A0A3E4A}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO.Pipelines.Performance.Tests", "tests\System.IO.Pipelines.Performance.Tests\System.IO.Pipelines.Performance.Tests.csproj", "{E68CF803-CC9F-458A-B0C7-1C2E0A0A3E4A}"
```

Comment:
I think they are caused by new version of VS. I have already submitted a couple of PR after updating to new VS and every time I had to roll back the sln changes. I think it's time to update the sln fine, unless you guys see a reason not to?